include/fmt/format-inl.h

```diff
@@ -981,7 +981,7 @@ FMT_FUNC void fallback_format(Double v, buffer<char>& buf, int& exp10) {
     denominator <<= 1 - fp_value.e;
     lower.assign(1);
     if (shift != 0) {
-      upper_store.assign(1 << shift);
+      upper_store.assign(1ull << shift);
```

Comment:
The result must be 32-bit. Does changing `1` to `1u` works around the bogus warning?

Reply:
Unfortunately no, faliled with the similar warning:
`
format-inl.h(984,1): warning C4334: '<<': result of 32-bit shift implicitly converted to 64 bits (was 64-bit shift intended?)
`

Reply:
Maybe this fix will be suitable?
`
upper_store.assign((1ull << shift) & 0xFFFFFFFF);
`